### PR TITLE
[TR] PIM-4897: Mongo DB status in SF2 debug toolbar

### PR DESCRIPTION
**Pull request by rybus**
| Q                  | A
| ------------------ | ---
| Migration script?  | No
| Behats             | No
| Specs & Static     | Yes
| Changelog updated  | No

MongoDB isn't enabled and not required:
mongodb_not_enabled_not_required

MongoDB isn't enabled and is required (because nb of product value > 5M)
mongodb_not_enabled_required

MongoDB is enabled:
mongodb_enabled

**Note**: the logo is is not translucid so if you are not hovering the icon, you can see its square borders. It needs to be replaced but I couldn't find one at first.

### CODE REVIEW

File: spec/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollectorSpec.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+use Doctrine\ORM\QueryBuilder;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class DatabaseCollectorSpec extends ObjectBehavior
+{
+    /** @var int the max number of product value allowed before having to switch to MongoBD */
+    const MYSQL_PRODUCT_VALUE_LIMIT = 5000000;
```
Comment (sumbobyboys) [inline spec/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollectorSpec.php]: you could drop it, not really necessary :)

File: src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        $this->storageDriver = $storageDriver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function collect(Request $request, Response $response, \Exception $exception = null)
+    {
+        $this->data = [
+            'mongodb_enabled' => $this->isMongoDbEnabled(),
```
Comment (sumbobyboys) [inline src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php]: you can align this lines :)

File: src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        $this->entityManager = $entityManager;
+        $this->storageDriver = $storageDriver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function collect(Request $request, Response $response, \Exception $exception = null)
+    {
+        $this->data = [
```
Comment (sumbobyboys) [inline src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php]: please declare it

File: src/Akeneo/Bundle/StorageUtilsBundle/Resources/views/Collector/database.html.twig [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    {# This left-hand menu appears when using the full-screen profiler. #}
+    <span class="label">
+        <span class="icon"><img
+                    src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAAcCAQAAADVGmdYAAAAAmJLR0QA/4ePzL8AAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQffAxkBCDStonIVAAAAGXRFWHRDb21tZW50AENyZWF0ZWQgd2l0aCBHSU1QV4EOFwAAAHpJREFUOMtj3PWfgXRAuqZd/5nIsIdhVBPFmgqIjCuYOrJsYtz1fxuUOYER2TQID8afwIiQ8YIkI4TzCv5D2AgaWSuExJKMIDbA7EEVhQEWXJ6FKUY4D48m7HYU/EcWZ8JlE6qfMELPDcUJuEMPxvYazYTDWRMjOcUyAEswO+VjeQQaAAAAAElFTkSuQmCC"
+                    alt=""/></span>
+        <strong>Example Collector</strong>
+    </span>
+{% endblock %}
+
+{% block panel %}
```
Comment (sumbobyboys) [inline src/Akeneo/Bundle/StorageUtilsBundle/Resources/views/Collector/database.html.twig]: you could drop that :)

File: src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php (hunk 3 above)
Comment (rybus) [inline src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php]: done

File: src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php (hunk 2 above)
Comment (rybus) [inline src/Akeneo/Bundle/StorageUtilsBundle/Collector/DatabaseCollector.php]: done

**Comment by nidup** [comment]
PR on 1.4 please

**Pull request closed without merging**